week1/TodoList.html

```diff
@@ -7,36 +7,36 @@
     <link rel="stylesheet" href="style.css">
 </head>
 <body>
-    <header><h2>Todo list</h2></header>
+    <header>
+        <h2>Todo list</h2>
+    </header>
     <nav>
         <div>오늘만 보기</div>
         <div>내일만 보기</div>
         <div>함께 보기</div>
     </nav>
-    <section>
-        <section>
+    <main>
+        <section class="Section_left">
```

Comment:
여기서도 html [class 네이밍](https://uipac.com/451)규칙 지켜주세용!! 바꾼다면
```suggestion
<section class="left-section">
```